tests/page-objects/display/languages.po.js

```diff
@@ -0,0 +1,72 @@
+const helper = require('../../helper');
+const utils = require('../../utils');
+
+const submitButton = element(by.css('div.form-actions button.btn.btn-primary.ng-scope'));
+const addLanguageButton = element(by.css('i.fa.fa-plus'));
+const addLanguageModal = element(by.css('body.ng-scope.modal-open'));
+const languageCodeInput = element(by.model('language.code'));
+const languageNameInput  = element(by.model('language.name'));
+const languageSubmitButton  = element(by.css('a.btn.submit.ng-scope.ng-binding.btn-primary'));
+const applicationLink = element(by.css('i.fa.fa-fw.fa-home'));
+
+const goToLanguagesTab = async () => {
+  await browser.sleep(3000);
+  await browser.get(utils.getAdminBaseUrl() + 'display/languages');
+  await utils.resetBrowserNative(addLanguageButton);
+  await helper.waitElementToBeVisibleNative(addLanguageButton);
+};
+
+const openAddLanguageModal = async () => {
+  await helper.clickElementNative(addLanguageButton);
+  await helper.waitElementToBeVisibleNative(addLanguageModal);
+};
+
+const addNewLanguage = async (code, name) =>{
+  await helper.clickElementNative(addLanguageButton);
+  await helper.waitElementToBeVisibleNative(addLanguageModal);
+  await languageCodeInput.sendKeys(code);
+  await languageNameInput.sendKeys(name);
+  await languageSubmitButton.click();
+};
+
+const languageDisplayed = async (code) =>{
+  const languageDiv = element(by.css(`#locale-${code}`));
+  const languageName = await helper.getTextFromElementNative(languageDiv);
+  return languageName;
+};
+const defaultLanguageDropdown=element(by.css('#locale'));
+const setDefaultLanguage = async (language) => {
+  await helper.selectDropdownByText(defaultLanguageDropdown, language);
+  await helper.clickElementNative(submitButton);
+  await helper.waitElementToBeClickable(submitButton);
+};
+
+const outgoingLanguageDropdown=element(by.css('#locale-outgoing'));
+const setOutgoingMessageLanguage = async (language) => {
+  await helper.selectDropdownByText(outgoingLanguageDropdown, language);
+  await helper.clickElementNative(submitButton);
+  await helper.waitElementToBeClickable(submitButton);
+};
+
+const isLanguageSelected = async (selector, code) => {
+  await helper.clickElementNative(element(by.css(selector)));
```

Comment:
NIT: Prefer passing in an element vs a selector. That way when you make a call to the function you can read what the element is.

EX: `isLanguageSelected(languagePo.defaultLanguageDropdown, 'afr')` vs  `languagesPage.isLanguageSelected('#locale', 'afr')` which means you redfine '#locale' anywhere this might be used.